hibernate-core/src/main/java/org/hibernate/tool/schema/extract/internal/SequenceInformationExtractorMariaDBDatabaseImpl.java

```diff
@@ -56,7 +57,7 @@ public Iterable<SequenceInformation> extractMetadata(ExtractionContext extractio
 				if ( sequenceInfoQueryBuilder.length() > 0 ) {
 					sequenceInfoQueryBuilder.append( UNION_ALL );
 				}
-				sequenceInfoQueryBuilder.append( String.format( SQL_SEQUENCE_QUERY, sequenceName ) );
+				sequenceInfoQueryBuilder.append( String.format( SQL_SEQUENCE_QUERY, sequenceName, Identifier.toIdentifier( sequenceName ) ) );
 			}
```

Comment:
what about keeping `%1Ss` in `SQL_SEQUENCE_QUERY ` and do `String.format( SQL_SEQUENCE_QUERY,Identifier.toIdentifier( sequenceName ) `;?

Reply:
If we did that, then we would potentially read a sequence name `` `seq-1` `` i.e. with the quoting part of the string, which is not correct.

Reply:
so when the sequence is quoted we extract a not quoted name?

Reply:
Look at the SQL statement `SELECT '%1$s' as sequence_name, minimum_value, maximum_value, start_value, increment, cache_size FROM %2$s`, the first message format parameter is within a SQL string literal, within which we don't need quoting. We only need quoting for when we render the name as identifier.

Reply:
sorry I missed the `'%1$s` so rephrase my question cannot be `SELECT %1$s as sequence_name, minimum_value, maximum_value, start_value, increment, cache_size FROM %1$s`
and then do `String.format( SQL_SEQUENCE_QUERY,Identifier.toIdentifier( sequenceName ) ;` ?

Reply:
ok  it seems that my proposal is not working :(

Reply:
No, because quoting with `` ` `` implies that the quoted thing is an identifier. We want to select a string though.

Reply:
TBH I also don't see what the advantage of the alternative approach is/would be ?